### ISSUE TITLE
Add level A, AA or AAA to all WCAG links on each page

### ISSUE THREAD

**Issue by EmmaDawsonDev** [AccessibleForAll/AccessibleWebDev#384]
Currently WCAG links look like:
1.1.1 Non text content

I would like to include the WCAG level like this:
1.1.1 Non text content (Level A)

Links on all pages should be updated with the correct levels.

**Comment by abdullahi-dev-so** [comment]
Hello @EmmaDawsonDev  I’d like to work on this issue. Could you assign it to me?
